extensions/amp-video/validator-amp-video.protoascii

```diff
@@ -183,6 +183,10 @@ tags: {  # <amp-video> in amp-story
     name: "captions-id"
     requires_extension: "amp-story-captions"
   }
+  attrs: {
+    name: "volume"
+    value_regex: "^((0?\.[1-9]+)?|1(\.0*)?)$"
```

Comment:
@ampproject/wg-caching For context, we're trying to disallow `0` since we'd like to prevent the publisher from shipping an audio track when it's fully muted.

Reply:
Added comment above to explain rationale, also added some test cases